apps/api/src/app/core/portfolio-calculator.spec.ts

```diff
@@ -35,13 +35,23 @@ function mockGetValue(symbol: string, date: Date) {
     }
   } else if (symbol === 'AMZN') {
     return { marketPrice: 2021.99 };
+  } else if (symbol === 'MFA') {
+    if (isSameDay(parse('2010-12-31', DATE_FORMAT, new Date()), date)) {
```

Comment:
there should be a `parseDate` function which doesn't require the 2nd and 3rd parameter